frontend/src/metabase/components/Schedule/strings.ts

```diff
@@ -57,6 +59,7 @@ export const getScheduleStrings = () => {
     // The context is needed because 'hourly' can be an adjective ('hourly
     // rate') or adverb ('update hourly'). Same with 'daily', 'weekly', and
     // 'monthly'.
+    minutely: c("adverb").t`by the minute`,
```

Comment:
To help the translators out a bit more, I would add some detail to this context string. Maybe it could be "adverbial phrase" or "this modifies a verb like 'Check'".